staging/src/k8s.io/client-go/examples/in-cluster-client-configuration/README.md

```diff
@@ -54,5 +54,5 @@ the `kubectl run` command and then run:
 
     kubectl delete deployment demo
 
-[sa]: https://kubernetes.io/docs/admin/authentication/#service-account-tokens
+[sa]: https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens
```

Comment:
I would link to https://kubernetes.io/docs/concepts/security/service-accounts/#how-to-use now that we have a https://kubernetes.io/docs/concepts/security/service-accounts/ page.

Reply:
Alternative option: link to https://kubernetes.io/docs/tasks/run-application/access-api-from-pod/

Reply:
Thanks for the feedback, now as the pr is merged do you recommend opening an issue for the change or is it ok otherwise, as I initially thought the link should refer to the specific term.